### ISSUE TITLE
Customizing Views

### ISSUE THREAD

**Issue by iLearnIdentityServer** [IdentityServer/IdentityServer3#2441]
Hi,

I would like to customize IdentityServer3 views. I was following documentation and findout that one can ether customize DefaultViewService by adding css files through DefaultViewServiceOptions and customizing views by putting pages in template folder with or without "_"...

The other way is registering custom View Service.

Currently, for me the first option seems to be good enough, latter on I might move to custom View Service.

The problem that I have right now is that my css does not get loaded.
I tried examples, they I work, by for some reason in my project its not working.

Basically in one assembly I have an implementation of stores, services, and also subset of dynamic client registration (including issuing initial one time registration tokens, endpoint itself, signing software statements, and other stuf)... I use this assembly in another project that is windows service project. 

The template folder is required to be where the Win Service exe is, but I can not figure out where should I put my css.

What every I put in viewOptions.Stylesheets.Add, and later on check through chrome debugger is see this css but empty.

**Comment by brockallen** [comment]
I'd do this:

1) Check if it's included in the rendered HTML. If not, then your view service is not including it in the rendered HTML, or maybe your layout file is missing the {stylesheets} placeholder.

2) See if you're getting a 404. If so, then your web server is not serving up the file.

**Comment by iLearnIdentityServer** [comment]
Yes, that is the problem. I was thinking (expecting) that for css you are doing the same thing as for view. Load from certain folder.

Ok. I run owin file server, not everything is ok.

Cheers!
